webrtc-stats.html

```diff
@@ -1531,6 +1542,14 @@ <h2>
                 sender of this stream, an <code><a>RTCSenderAudioTrackAttachmentStats</a></code> or
                 <code><a>RTCSenderVideoTrackAttachmentStats</a></code>.
               </dd>
+              <dt>
+                <dfn><code>mediaSourceId</code></dfn> of type <span class=
+                "idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                The identifier of the stats object representing the track currently attached to the
+                sender of this stream, an <code><a>RTCMediaSourceStats</a></code>.
```

Comment:
I'm worried about having stats defined because of the present state of our implementation, instead of where they logically belong. How many people will implement source stats before sender stats?
And what about track stats? If we're putting this in for backwards compatibility, shouldn't the track get one too?

Reply:
I was planning to implement source stats before sender stats in Chrome. Having access to resolution prior to encoding is a "migration blocker" that I want done for M76. Moving "track" stats to "sender" is less important right now. (And moving "track" stats to "outbound-rtp" is mostly important for simulcast, not migration either)

The "track" stats get mediaSourceId already in this PR, they're inherited from RTCVideoSenderStats and RTCAudioSenderStats to RTCSenderVideoTrackAttachmentStats and RTCSenderAudioTrackAttachmentStats.